src/Tribe/Views/V2/View.php

```diff
@@ -581,8 +591,12 @@ public function get_url( $canonical = false ) {
 			}
 		}
 
-		// When we find nothing we're always on page 1.
-		$page = $this->repository->count() > 0 ? $this->url->get_current_page() : 1;
+		if( $this->should_reset_page() ){
```

Comment:
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 5e6ab5a62bbe6d3b78b522121cc7f408</sub>